server/config/db.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    HOST: 'localhost',
+    USER: "root",
+    PASSWORD: "maffmann",
+    DB: "mentel_DB",
+    dialect: "mysql",
+    pool: {
+        max: 5,
+        min: 0,
+        acquire: 30000,
+        idle: 10000
+    }
```

Comment:
use environment variables to avoid exposing your credentials
check `https://www.npmjs.com/package/dotenv`

Reply:
ok..